### PR TITLE
feat: Add popular collections on AHK to sidebar on explore

### LINKED ISSUE #7131: Add popular collections on AHK to sidebar on explore
Best in alphabetical order

- https://canary.kodadot.xyz/ahk/collection/u-2024
- https://canary.kodadot.xyz/ahk/collection/u-2023
- https://canary.kodadot.xyz/ahk/collection/110/activity
- https://canary.kodadot.xyz/ahk/collection/30/activity
- https://canary.kodadot.xyz/ahk/collection/14/activity
- https://canary.kodadot.xyz/ahk/collection/u-117/activity
- https://canary.kodadot.xyz/ahk/collection/u-2022
- https://canary.kodadot.xyz/ahk/collection/u-31848

image

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

  👇 __ Let's make a quick check before the contribution.

  ## PR Type

  - [x] Bugfix
  - [x] Feature
  - [ ] Refactoring

  ## Needs QA check

  - @kodadot/qa-guild please review

  ## Context

  - [x] Closes #7131
  - [ ] Requires deployment <snek/rubick/worker>

  #### Did your issue had any of the "$" label on it?

  - [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)

  ## Screenshot 📸

  - [x] My fix has changed UI

image

  ## Copilot Summary
  
### <samp>🤖 Generated by Copilot at ce5fd6e</samp>

Added `ahk` chain to popular collections filter and improved its UI. Removed unused code and simplified the display of collection chain names using CSS.

  
### <samp>🤖 Generated by Copilot at ce5fd6e</samp>

> _We filter the collections of doom_
> _We choose the `ahk` chain of gloom_
> _We purge the code of useless waste_
> _We style the names with CSS grace_

### CODE REVIEW

File: components/shared/filters/modules/constants.ts [hunk 1]
```diff
@@ -43,4 +43,5 @@ export const POPULAR_COLLECTIONS = {
     '2334273319', //Guardians and Creatures
     '1635680444', //KODACHAINS
   ],
+  ahk: ['u-2024', 'u-2023', '110', '30', '14', 'u-117', 'u-2022', 'u-31848'],
```
Comment (roiLeo) [inline components/shared/filters/modules/constants.ts:46]: I would add comments with description on collection name

Comment (Jarsen136) [reply components/shared/filters/modules/constants.ts:46]: It's not needed because we have do sort for the collection by their names.

Comment (roiLeo) [reply components/shared/filters/modules/constants.ts:46]: I want to know which collection corresponds to these ids when I look at the code, please make it like other chains 🥺

Comment (Jarsen136) [reply components/shared/filters/modules/constants.ts:46]: ok, will update it

**Review by roiLeo — APPROVED** [review]
Thanks gtg

**Review by daiagi — APPROVED** [review]

**Review by stephenjason89 — APPROVED** [review]
Looks good 😊

**Comment by yangwao** [comment]
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 4.27 USD/DOT ~ 7.026 $DOT
🧗 [16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z](https://polkadot.subscan.io/account/16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)
🔗 [0x7b9d2176e45609d0e15e2be00a8781472ea5938f092a8050d23265899734a3b2](https://polkadot.subscan.io/extrinsic/0x7b9d2176e45609d0e15e2be00a8781472ea5938f092a8050d23265899734a3b2)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**